oak_functions/loader/src/tests.rs

```diff
@@ -258,11 +279,21 @@ fn parse_lookup_entries_invalid() {
 async fn lookup_data_refresh() {
     let mock_static_server = Arc::new(test_utils::MockStaticServer::default());
 
+    let (terminate_static_server_tx, terminate_static_server_rx) =
+        tokio::sync::oneshot::channel::<()>();
+
+    let static_server_port = test_utils::free_port();
     let mock_static_server_clone = mock_static_server.clone();
-    tokio::spawn(async move { mock_static_server_clone.serve(STATIC_SERVER_PORT).await });
+    let static_server_join_handle = tokio::spawn(async move {
+        mock_static_server_clone
+            .serve(static_server_port, async {
+                let _ = terminate_static_server_rx.await;
```

Comment:
nit: Could this be `terminate_static_server_rx.await.unwrap()` like line 144 above? To be consistent.

Reply:
Done.